test/e2e/configuration_policy_test.go

```diff
@@ -214,10 +201,7 @@ var _ = Describe("Test configuration policy inform", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			common.DoRootComplianceTest(rolePolicyName, policiesv1.Compliant)
 			expectedStatusMsgs = append(
-				[]string{
-					"Compliant; notification - roles [role-policy-e2e] in namespace default missing " +
-						"as expected, therefore this Object template is compliant",
-				},
+				[]string{"Compliant; notification - roles [role-policy-e2e] missing as expected in namespace default"},
 				expectedStatusMsgs...,
```

Comment:
is this `mustnothave` just double check

Reply:
That's right. It is inform and `mustnothave`.